File: python_project/oops/Banking/CommercialCustomer.py
from Customer import Customer 

class CommercialCustomer(Customer):
    def __init__(self, name: str):
        super().__init__(name)  
        self.__address = None
        self.__contact_person = None
        self.__contact_person_phone = None

    def get_address(self) -> str:
        return self.__address

    def set_address(self, address: str) -> None:
        self.__address = address

    def get_contact_person(self) -> str:
        return self.__contact_person

    def set_contact_person(self, contact_person: str) -> None:
        self.__contact_person = contact_person

    def get_contact_person_phone(self) -> str:
        return self.__contact_person_phone

    def set_contact_person_phone(self, contact_person_phone: str) -> None:
        self.__contact_person_phone = contact_person_phone

File: java_project/oops/CashForMetals/CommercialCustomer.java
package oops.CashForMetals;


public class CommercialCustomer extends Customer {
    private String address;
    private String contactPerson;
    private String contactPersonPhone;

    public CommercialCustomer(String name) {
        super(name); // Takes 1 String [cite: 29]
    }

    public String getAddress() { return address; }
    public void setAddress(String address) { this.address = address; }

    public String getContactPerson() { return contactPerson; }
    public void setContactPerson(String contactPerson) { this.contactPerson = contactPerson; }

    public String getContactPersonPhone() { return contactPersonPhone; }
    public void setContactPersonPhone(String contactPersonPhone) { this.contactPersonPhone = contactPersonPhone; }

    @Override
    public String toString() {
        return super.toString() + " (Commercial) | Contact: " + contactPerson;
    }
}
